namespace csharp Ruyi.SDK.StorageLayer
namespace cpp Ruyi.SDK.StorageLayer

struct GetLocalPathResult {
    1: bool result,
    2: string path,
}


